src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/SlimefunLocalization.java

```diff
@@ -215,76 +215,76 @@ public ItemStack getRecipeTypeItem(Player p, RecipeType recipeType) {
     }
 
     @Override
-    public void sendMessage(CommandSender sender, String key, boolean addPrefix) {
+    public void sendMessage(CommandSender recipient, String key, boolean addPrefix) {
```

Comment:
Its weird because the variable name `recipient` for some reason sounds wrong in this context (same for all the other uses in this file), and I would then suggest something like "endUser" but it literally means the same thing, guess my mind is just acting weird xD

Reply:
Yeah, it sounds weird and it is more accurate. `sender` had different implications, so despite this sounding weird it is more accurate.